google/cloud/spanner/samples/samples.cc

```diff
@@ -527,6 +527,126 @@ void FieldAccessOnNestedStruct(google::cloud::spanner::Client client) {
 }
 //! [END spanner_field_access_on_nested_struct]
 
+void Send(std::string) {}  // NOLINT
+
+void SendPartitionToRemoteMachine(
+    google::cloud::spanner::ReadPartition const& partition) {
+  //! [serialize-read-partition]
+  namespace spanner = google::cloud::spanner;
+  google::cloud::StatusOr<std::string> serialized_partition =
+      spanner::SerializeReadPartition(partition);
+  if (serialized_partition.ok()) {
+    Send(*serialized_partition);
```

Comment:
(optional) FWIW, I'd write things like this with a comment instead of a dummy `Send()` function. Maybe something like

```cc
if (serialized_partition) {
  // Use *serialized_partition, write to disk, send to another machine, etc.
}
```
I think that comment would convey more info than `Send()` and it might avoid the need to NOLINT.

Reply:
I am going to suggest that we need something more than that.  How do I "save" one of these serialized partitions?  Is the string safe to save and read via iostreams?  Does it contain non-ASCII characters? What about NUL characters?

It would be nice to show how would applications store the string somewhere using standard C++.